com.reprezen.swagedit.openapi3/resources/templates.xml

```diff
@@ -0,0 +1,328 @@
+<templates>
+	<template name="OpenApi version" 
+       id="com.reprezen.swagedit.openapi3.templates.openapi_template" 
+       description="OpenAPI version declaration" 
+       context="com.reprezen.swagedit.openapi3.templates.root" 
+       enabled="true">openapi: "3.0"</template>
+
+<!--
+	INFO  
+ -->
+
+	<template name="OpenApi info" 
+       id="com.reprezen.swagedit.openapi3.templates.info_template" 
+       description="Provides descriptive metadata about this API specification" 
+       context="com.reprezen.swagedit.openapi3.templates.root" 
+       enabled="true">info:
+  version: 1.0
```

Comment:
I suggest changing `1.0` -> `"1.0.0"`
Three of four available v3 examples use semantic versioning for info.version ("1.0.0"), and only one uses "v2".
Also, validation thinks that the info.version in its current form is a number and shows an error
> value of type number is not allowed, value should be of type string